lib/dynflow/throttle_limiter.rb

```diff
@@ -0,0 +1,105 @@
+module Dynflow
+  class ThrottleLimiter
```

Comment:
It would only make sense if the interface would match the
DelayedExecutor::Abstract (could be updated on both
DelayedExecution::Abstract and the TrhottleLimiter), but only if it was
obvious on how to do so. It can always be done later.

On 1 February 2016 at 15:55, adamruzicka <notifications@github.com> wrote:

> In lib/dynflow/throttle_limiter.rb
> <https://github.com/Dynflow/dynflow/pull/179#discussion_r51426802>:
>
> > @@ -0,0 +1,105 @@
> > +module Dynflow
> > +  class ThrottleLimiter
>
> Wouldn't it be odd to have something like DelayedExecutor::ThrottleLimiter
> which would not extend DelayedExecutor::Abstract? However I wouldn't mind
> moving it from the ::Dynflow scope
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Dynflow/dynflow/pull/179/files#r51426802>.
>